packages/opentelemetry-sdk-trace-base/src/export/BatchSpanProcessorBase.ts

```diff
@@ -136,7 +136,11 @@ export abstract class BatchSpanProcessorBase<T extends BufferConfig>
     }
 
     this._finishedSpans.push(span);
-    this._maybeStartTimer();
+    if (this._finishedSpans.length == this._maxExportBatchSize) {
```

Comment:
maybe `>=` ?
not sure if it always adds them 1 by 1 but just to make sure

Reply:
Putting `>=` here is problematic because it may trigger multiple batches, before the first batch drains elements from `_finishedSpans` bringing its size below the `_maxExportBatchSize`.

They are always added one by one with `this._finishedSpans.push(span);` in line 138, so I think `==` should work fine.